src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/FhirJsonInputFormatter.cs

```diff
@@ -56,6 +58,25 @@ public override async Task<InputFormatterResult> ReadRequestBodyAsync(InputForma
 
             var request = context.HttpContext.Request;
 
+            _parser.Settings.AllowUnrecognizedEnums = false;
```

Comment:
var newsettings = _parser.Settings.Clone()
new FhirJsonParser(newsettings)

Reply:
@brendankowitz  as we discussed, I cloned settings of existing parser and created a new parser just for bundles. I verified parser in other places are not impacted.